### PR TITLE
Add ability to copy/paste messages #639

### LINKED ISSUE #639: Add ability to copy/paste messages
There is currently no way to copy a message or a passphrase.

Reports:
https://instabug.com/applications/status-10bb8cbc-15f6-44f0-8c7a-db478dc34d7d/beta/bugs/85
https://instabug.com/applications/status-10bb8cbc-15f6-44f0-8c7a-db478dc34d7d/beta/bugs/89
https://instabug.com/applications/status-10bb8cbc-15f6-44f0-8c7a-db478dc34d7d/beta/bugs/123

### PR DESCRIPTION
**Pull request by GustavoNunes**
Overview: reused the same function from the profile screen (which copies the public key)

- I've made it work only on the text messages, not command ones
- It's copying the raw content, so text decoration is preserved. Maybe for the future, another option could be added to copy just the text?

**Implementation:**
Both the share function and the event handler were in the _profile_ namespace, although they seemed more general to me.
- Moved the share function to the existent component/share namespace.
- Moved the share event handler to the root handlers file (couldn't figure out a better place)
- Added the share action to the message component (not sure if I choose the best place) and cleared namespace declarations everywhere.

fixes #639

### CODE REVIEW

**Comment by jarradh** [comment]
please pull into `develop`, not `master`, thanks :)

**Comment by jarradh** [comment]
ah theres conflicts in this PR now from other merge :)

**Comment by GustavoNunes** [comment]
solved @jarradh :)

File: src/status_im/handlers.cljs [hunk 1]
```diff
@@ -42,6 +45,18 @@
   (fn [db [_ k v]]
     (assoc-in db [:animations k] v)))
 
+(register-handler :open-sharing
+                  (u/side-effect!
```
Comment (alwx) [inline src/status_im/handlers.cljs]: The spacing here is too big. I think it is better to make it look the way it looked before.

**Comment by alwx** [comment]
1. Do we really need to allow user to open sharing dialog when tapping on a message? If you take a look at default messaging app or messengers like Telegram, you will see that it is possible to copy a message there, but it is impossible to "share" it. 
2. Isn't it better to open the dialog on long press? This is, once again, the way it works in other apps.

**Comment by jarradh** [comment]
1. Ideally we could bring up the OS text tools, unsure how best to do this

2. Long press is ideal and more consistent

**Comment by GustavoNunes** [comment]
Thanks guys! Gotta adjust emacs indentation...I fixed that and changed to long press. Didn't change the share option though

**Pull request merged**